README.md

```diff
@@ -48,15 +51,28 @@ mount_svfs myName /mountpoint -o username=..,password=..,tenant=..,region=..,con
 - Usage with OVH Public Cloud Storage is explained [here](docs/PCS.md).
 - Usage with hubiC is explained [here](docs/HubiC.md).
 
+
 ## FAQ
 
+
 Got errors using `rsync` with svfs ? Can't change creation time ? Why svfs after all ?
 
 Take a look at the [FAQ](docs/FAQ.md).
 
 
+## Commands 
+
+* `info`: Display some information about SVFS current version and check if a new version of SVFS exist.
+
+
 ## Options
 
+#### Mandatory options
```

Comment:
Depending on your usage (pcs/hubic) some options are mandatory and some aren't, so not only device and mountpoint are required. Rename this to *"Device"* options instead ?